lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -70,9 +70,9 @@
                                 emailAttribute: 'email',
                                 linkTitle: gettext('Reset Your Password'),
                                 linkHref: fieldsData.password.url,
-                                helpMessage: gettext('When you click "Reset Your Password", edX will send a message ' +
-                                    'to the email address for your edX account. Click the link in the message to ' +
-                                    'reset your password.')
+                                helpMessage: window.interpolate_text(
+                                    gettext('When you click "Reset Your Password", {platform_name} will send a message to the email address for your {platform_name} account. Click the link in the message to reset your password.'), {platform_name: platformName}    /* jshint ignore:line */
```

Comment:
I would say:

``When you select "Reset Your Password", a message will be sent to the email address for your {platform_name} account. Click the link in the message to reset your password.``